From: Hilko Bengen <bengen@debian.org>
Date: Sun, 12 Feb 2012 18:05:49 +0100
Subject: Eliminate duplicate entries from package list

"apt-cache depends --recurse -i ..." on Ubuntu 12.04 (apt
0.8.16~exp12ubuntu3) outputs duplicate entries which broke the code
that checks whether all package files have been successfully
downloaded. I am not sure if this is a bug in apt.
---
 src/febootstrap_debian.ml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/febootstrap_debian.ml b/src/febootstrap_debian.ml
index 58e1b15..8562a7a 100644
--- a/src/febootstrap_debian.ml
+++ b/src/febootstrap_debian.ml
@@ -53,7 +53,7 @@ let rec debian_resolve_dependencies_and_download names =
     if Config.apt_cache_depends_recurse_broken then
       workaround_broken_apt_cache_depends_recurse (sort_uniq pkgs)
     else
-      pkgs in
+      sort_uniq pkgs in
 
   (* Exclude packages matching [--exclude] regexps on the command line. *)
   let pkgs =
